nightwatch.config.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
```

Comment:
I think some comments in here describing what everything is doing could be helpful. Not being super familiar with CircleCI, some of this looks like magical incantation voodoo.